src/processing_spec.hpp

```diff
@@ -19,12 +19,19 @@
 
 namespace ovms {
 
-struct SequenceProcessingSpec {
-    const uint32_t sequenceControlInput;
-    const uint64_t sequenceId;
+class SequenceProcessingSpec {
+    uint32_t sequenceControlInput;
```

Comment:
We do not need const no more ?

Reply:
Because now it's a private member and we provide public getter and setter for it.